Графика для всех страниц

KateSolodchuk opened the pull request:
Экспортировала из Фигмы все изображения, фоновые изображения для форм и звездочки пока не вставляла в проект, решила, что все же вставлю их средствами CSS. 

---
:mortar_board: [Графика для всех страниц](https://up.htmlacademy.ru/htmlcss-individual/1/user/2357681/tasks/5)

:boom: https://htmlacademy-htmlcss.github.io/2357681-sedona-1/3/

KateSolodchuk committed:
Исправляет логотип в шапке

Исправлен путь к логотипу в шапке и убрана ссылка на главную в логотипе в шапке главной страницы (чтобы главная не перезагружалась при клике)

KateSolodchuk committed:
Окончательно исправляет логотип в шапке

Исправлено название папки с картинками и путь к логотипу в шапке, добавлены размеры

KateSolodchuk committed:
Добавляет папку с графикой и добавляет графику в разметку навигации

KateSolodchuk committed:
Добавляет картинки в разметку главной страницы

Все, кроме размытого фона в форме подписи на рассылку

KateSolodchuk committed:
Добавляет картинки в разметку каталога

Кроме фона формы и звездочек рейтинга

KateSolodchuk committed:
Исправляет формат иконок соцсетей, преимуществ и логотипы Седоны и HTML Academy

KateSolodchuk committed:
Доделывает изображения в разметке, исправляет значение width на целые числа

Director099 reviewed (commented):
Замечание по единообразию
1) http://joxi.ru/nAyd5gXckEM4x2 Все картинки должны начинаться с маленькой буквы т.к. и и конки тоже с маленькой
2) в папке list название иконок пишешь через `=` соблюдай единообразие это нижнее подчеркивание `_`

В идеале дефис `-` Его использует академия в нейминге картинок

Director099 commented on catalog.html:
```diff
@@ -39,14 +41,17 @@
-          <a class="breadcrumbs-link" href="index.html"><img class="breadcrumbs-img" src="#" alt="Главная страница"></a>
+          <a class="breadcrumbs-link" href="index.html"><img class="breadcrumbs-img" src="images/icons/breadcrumbs/breadcrumbs_home.svg" alt="Главная страница"></a>
+        </li>
+        <li class="breadcrumbs-item">
+          <img src="images/icons/breadcrumbs/breadcrumbs_arrow.svg" width="6" height="10" alt="Далее">
         </li>
```
Это li лишнее, стрелка рисуется декором потому оно не часть списка

KateSolodchuk committed:
Исправляет замечания

Поправила названия картинок (в качестве разделителя пока оставила нижнее подчеркивание, потом в процессе работы над третьим блоком переименую тогда и оставлю везде дефисы, надо внимательно все проверить будет), убрала элемент списка со стрелкой в хлебных крошках
